bgs/handlers.go

```diff
@@ -155,9 +176,74 @@ func (s *BGS) handleComAtprotoSyncListBlobs(ctx context.Context, cursor string,
 }
 
 func (s *BGS) handleComAtprotoSyncListRepos(ctx context.Context, cursor string, limit int) (*comatprototypes.SyncListRepos_Output, error) {
-	return nil, fmt.Errorf("NYI")
+	if limit > 1000 {
+		limit = 1000
+	}
+
+	// Use UIDs for the cursor
+	var err error
+	c := int64(0)
+	if cursor != "" {
+		c, err = strconv.ParseInt(cursor, 10, 64)
+		if err != nil {
+			return nil, fmt.Errorf("invalid cursor: %w", err)
+		}
+	}
+
+	users := []User{}
+	if err := s.db.Model(&User{}).Where("uid > ?", c).Limit(limit).Find(&users).Error; err != nil {
```

Comment:
does limit actually do what we want here without an 'order by'?

Reply:
o yea good catch